include/vao.hpp

```diff
@@ -20,6 +20,6 @@ class VAO {
   void bind();
   static void unbind();
   void deleteArray();
-  void linkVBO(const VBO &vbo, uint8 slot, uint8 n_components, GLenum type,
+  void linkVBO(VBO &vbo, uint8 slot, uint8 n_components, GLenum type,
```

Comment:
Could you share what the error was?
it's weird, since on my end it is fine and well, it is preferred to be flagged as "const".

Reply:
I added this back to the method and seems like there are no errors this time